pinterest/teletraan
# Reset environment variables in between script runs

Problem:
---
Deployd sets environment variables for current deploy goal. If there is interliving deploys or if deploy goal changes, environment variables could become stale and reused.

Fix:
---
Reset environment variables in between script runs.

Test:
---
1. Verify script variables are propagated for given deploy.
2. Verify script variables are cleared when removed from Teletraan UI.
3. Deploy service A and B back-to-back and make sure deploy of B does not use environement variables from service A.
